test/functional/feature_block.py

```diff
@@ -169,7 +169,7 @@ def run_test(self):
         self.log.info("Reject a block where the miner creates too much coinbase reward")
         self.move_tip(6)
         b9 = self.next_block(9, spend=out[4], additional_coinbase_value=1)
-        self.sync_blocks([b9], False, 16, b'bad-cb-amount', reconnect=True)
+        self.sync_blocks([b9], False, 'coinbase pays too much (actual=10000000000 vs limit=9999999999)', reconnect=True)
```

Comment:
Would be nice if all calls to `sync_blocks` used named argument for `reject_reason`, but that change should be in a future PR.